README.md

```diff
@@ -84,6 +84,32 @@ Creating a CSV export for Carbonite Migration
     1. Enter the location of the CSV file generated on D42 website. The source machine IP will be parsed from this file.
     2. The "Carbonite target IP" is the replication server and the server added to the Carbonite console application. This machine stores the protected data.
     3. The "ESX host IP/HyperV host IP" is the ESXi/HyperV VM Server which will host the replica of your source machine.
+    4. When asked for the type of job you will be creating, please be aware that the available choices differ for Windows and Linux OSs. The following jobs are available: 
+
+        **Availability for Windows jobs:**
+        - **FilesAndFolders** — Files and folders
```

Comment:
I think we should list only the jobs that we tested with, so just the Windows and Linux move jobs.

Reply:
Ok, I'll update that.